### ISSUE TITLE
Error on log file parsing after recent release

### ISSUE THREAD

**Issue by michaelpacer** [SublimeText/LaTeXTools#802]
I got an error saying that 

```
LaTeXTools could not parse the TeX log file …
(actually, we never should have gotten here)
```

Looked into the console and got:

```python
Traceback (most recent call last):
  File "/Users/cocosci/Library/Application Support/Sublime Text 3/Packages/LaTeXTools/makePDF.py", line 289, in run
    data, self.caller.tex_dir
TypeError: parse_tex_log() takes 1 positional argument but 2 were given
```

**Comment by ig0774** [comment]
That should go away if you exit and then restart ST. Please let me know if that isn't the case.

**Comment by HughParsonage** [comment]
I also get it following the upgrade.

```tex
\documentclass{article}

\begin{document}
Hello. 
\end{document}
```

Log file

```
parseTeXlog: Parsing log file
parseTeXlog: FILE? Line:This is pdfTeX, Version 3.14159265-2.6-1.40.17 (MiKTeX 2.9 64-bit) (preloaded format=pdflatex 2016.5.26)  6 AUG 2016 18:18
parseTeXlog: MATCHED
parseTeXlog: FILE? Line:entering extended mode
parseTeXlog: FILE? Line:**./barebones.tex
parseTeXlog: FILE? Line:(barebones.tex
parseTeXlog: MATCHED
parseTeXlog: IT'S A FILE!
parseTeXlog:  C:\Users\hugh\Documents\sandbox\barebones.tex (4)
parseTeXlog: Reprocessing 
parseTeXlog: FILE? Line:Babel <3.9r> and hyphenation patterns for 75 language(s) loaded.
parseTeXlog: FILE? Line:("C:\Program Files\MiKTeX 2.9\tex\latex\base\article.cls"
parseTeXlog: MATCHED
parseTeXlog: FILE? Line:("C:\Program Files\MiKTeX 2.9\tex\latex\base\size10.clo"
parseTeXlog: MATCHED
parseTeXlog:  C:\Users\hughGrattan\Documents\sandbox\barebones.tex (11)
parseTeXlog: Popped file: C:\Users\hughGrattan\Documents\sandbox\barebones.tex (11)
parseTeXlog: Reprocessing 
parseTeXlog: PERR [')' no files]
False True True
Jump to:  3 7
<class '_latextools_sumatra_viewer.SumatraViewer'>
```

**Comment by ig0774** [comment]
@HughParsonage: Could you post the actual log file for that, e.g., `barebones.log`?

**Comment by HughParsonage** [comment]
```
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (MiKTeX 2.9) (preloaded format=pdflatex 2016.6.28)  7 AUG 2016 00:26
entering extended mode
**"C:/Users/Hugh Parsonage/Documents/sandbox/barebones.tex"
("C:/Users/Hugh Parsonage/Documents/sandbox/barebones.tex"
LaTeX2e <2016/03/31> patch level 1
Babel <3.9r> and hyphenation patterns for 75 language(s) loaded.
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\base\article.cls"
Document Class: article 2014/09/29 v1.4h Standard LaTeX document class
("C:\Program Files (x86)\MiKTeX 2.9\tex\latex\base\size10.clo"
File: size10.clo 2014/09/29 v1.4h Standard LaTeX file (size option)
)
\c@part=\count79
\c@section=\count80
\c@subsection=\count81
\c@subsubsection=\count82
\c@paragraph=\count83
\c@subparagraph=\count84
\c@figure=\count85
\c@table=\count86
\abovecaptionskip=\skip41
\belowcaptionskip=\skip42
\bibindent=\dimen102
) (barebones.aux)
\openout1 = `barebones.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
 [1

{C:/Users/Hugh Parsonage/AppData/Local/MiKTeX/2.9/pdftex/config/pdftex.map}] (b
arebones.aux) ) 
Here is how much of TeX's memory you used:
 203 strings out of 493335
 2268 string characters out of 3136651
 53225 words of memory out of 3000000
 3817 multiletter control sequences out of 15000+200000
 3640 words of font info for 14 fonts, out of 3000000 for 9000
 1141 hyphenation exceptions out of 8191
 23i,4n,17p,161b,107s stack positions out of 5000i,500n,10000p,200000b,50000s
<C:/Program Files (x86)/MiKTeX 2.9/fonts/type1/public/amsfonts/c
m/cmr10.pfb>
Output written on barebones.pdf (1 page, 11558 bytes).
PDF statistics:
 10 PDF objects out of 1000 (max. 8388607)
 0 named destinations out of 1000 (max. 500000)
 1 words of extra memory for PDF output out of 10000 (max. 10000000)

```

**Comment by ig0774** [comment]
Thanks for the issue reports. Both of these issues should be resolved by v3.10.5.
